src/components/forms/Combobox.tsx

```diff
@@ -339,8 +341,10 @@ export function ComboboxAsync<T = any>({
           response.data.data.map((entry) =>
             data.push({
               id: entry[entryOptions.id],
-              label: entry[entryOptions.label],
-              value: entry[entryOptions.value],
+              label:
+                entryOptions.formatLabel?.(entry) ?? entry[entryOptions.label],
+              value:
+                entryOptions.formatValue?.(entry) ?? entry[entryOptions.value],
```

Comment:
This is generally a bad idea. Can you let me know the reasons why we want to format the value? It's generally a pitfall. Looking at the current tax rate selector, I don't see any formatting of value.

Reply:
@beganovich The reason for doing this is actually specific `defaultValue` that should be passed in the `TaxRateSelector` component for example `${expense.tax_name1}${expense.tax_rate1}`. If we do not have formatted value when we pass this specific defaultValue nothing will be selected because the default value is `rate` property.

The actual part of the code where I did value formatting is this one: 

`formatValue: (taxRate) => ${taxRate.name}${taxRate.rate}`

I did label formatting because of the reason that I explained in the previous comment dicussion.

Actually I have some other ideas for formatting this but I thought that this one is the most clear and provide us better adjusting values.

Let me know your thoughts.